packages/block-library/src/list/ordered-list-settings.native.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { InspectorControls } from '@wordpress/block-editor';
+import { TextControl, PanelBody, ToggleControl } from '@wordpress/components';
+
+function limitIntegerValue( value ) {
```

Comment:
Since we're not taking in a min / max and it's specific to signed 32-bit integers, should we make the function name more specific, or add a comment describing what it does?

Reply:
IMO a comment is fine if we're [keeping it local to these files](https://github.com/WordPress/gutenberg/pull/47362#discussion_r1132774553).